.github/workflows/rust.yml

```diff
@@ -1,46 +1,67 @@
-name: Build
+name: ion-rust CI Build
```

Comment:
This name is the name and URL of the badge, so whatever it is will have to be changed in the two README files (and we have to be happy with the text).  Personally, if we're going to change the text `CI Build` is better, or `Ion Rust CI Build` just to be case consistent and keep the badge label reasonable.

![image](https://user-images.githubusercontent.com/503506/89788246-b0f4a880-dad3-11ea-9798-5433e138bbf3.png)

Reply:
I forgot about the badge. I will update the name.

Reply:
We have two badges, FYI for the READMEs in both Crates.  I did this because when we publish `ion-c-sys`, I wanted it to also have a reasonable README.